content/changelog/2024/cody-promoted-prompts.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Promoted Prompts for VS Code and Cody Web"
+authors:
+  - name: Kelvin Yap
+    url: https://x.com/kelvinyap
+    avatar: https://storage.googleapis.com/sourcegraph-assets/blog/kelvin_avatar.png
+publishDate: 2024-10-30T17:34-07:00
+description: "Promoted Prompts elevates Prompts to Cody users to the top of the Prompts list or Prompt Library, allowing organizations to highlight specific Prompts that encourage best practices or are recommended to use."
+tags: [Enterprise, Cody, VS Code, Cody Web]
+version: [5.9.0]
+versionIcon: /assets/changelog/sourcegraph.svg
+slug: 'cody-promoted-prompts'
+published: true
+
+---
+
+The Prompt Library is a great way for users within an organization to share, discover, and use prompts while working with Cody Chat. We’ve introduced a new concept to Prompts called Promoted Prompts which allows admins to highlight Prompts to users that appear at the top of a Prompt list, directing users to use specific Prompts that encourage best practices within the organization.
+
+Admins can promote a Prompt by checking the `Promoted` box in the Prompt edit screen.
```

Comment:
@kelvinyap is the new decision to capitalize every usage of "Prompt," even when referring to individual ones?